src/status_im/subs/wallet/collectibles.cljs

```diff
@@ -91,35 +90,36 @@
              current-account-collectibles))))
 
 (re-frame/reg-sub
- :wallet/last-collectible-details
- :<- [:wallet]
- (fn [wallet]
-   (let [last-collectible (:last-collectible-details wallet)]
-     (assoc last-collectible :preview-url (preview-url last-collectible)))))
+ :wallet/collectible
+ :<- [:wallet/ui]
+ :-> :collectible)
 
 (re-frame/reg-sub
- :wallet/last-collectible-aspect-ratio
- :<- [:wallet]
- (fn [wallet]
-   (:last-collectible-aspect-ratio wallet)))
+ :wallet/collectible-details
+ :<- [:wallet/collectible]
+ (fn [collectible]
+   (as-> collectible $
+     (:details $)
+     (assoc $ :preview-url (preview-url $)))))
 
 (re-frame/reg-sub
- :wallet/last-collectible-details-chain-id
- :<- [:wallet/last-collectible-details]
+ :wallet/collectible-aspect-ratio
+ :<- [:wallet/collectible]
  (fn [collectible]
-   (get-in collectible [:id :contract-id :chain-id])))
```

Comment:
This subscription is not needed, it is just working as an extractor, and a deep one (because of `get-in` these extractions are only being used in a component that already has access to the `collectible`, so using these subs makes the code harder to track